client/components/jetpack/backup-card/style.scss

```diff
@@ -131,7 +174,7 @@ $cloud-icon-height: 24px;
 
 	> li {
 		display: flex;
-		align-items: flex-start;
```

Comment:
This needs to stay `flex-start` for the successful backup state. If the avatar is centered it can make the list more difficult to read when there are several items:
<img width="490" alt="Screen Shot 2020-10-21 at 1 54 09 PM" src="https://user-images.githubusercontent.com/1620183/96760480-ec9ad100-13a6-11eb-92c1-2455769fe6e0.png">
<img width="466" alt="Screen Shot 2020-10-21 at 2 10 19 PM" src="https://user-images.githubusercontent.com/1620183/96761612-2e2b7c00-13a7-11eb-9731-c166c6eb3e21.png">

Reply:
Yes, makes sense. Good catch, thanks!  Fixed 👍